bindings/cs/MongoDB.Crypt/LocalKeyId.cs

```diff
@@ -19,15 +19,15 @@ namespace MongoDB.Crypt
     /// <summary>Contains all the information needed to describe a Local KMS CMK.</summary>
     public class LocalKeyId : IKmsKeyId, IInternalKmsKeyId
     {
-        public KmsType KeyType => KmsType.Local;
-
         /// <summary>
-        /// Gets or sets the key.
+        /// Creates an <see cref="LocalKeyId"/> class.
         /// </summary>
-        /// <value>
-        /// The key.
-        /// </value>
-        public byte[] Key { get; set; }
+        /// <param name="key">The key.</param>
+        public LocalKeyId()
```

Comment:
I'm not clear what this class is for.  It no longer has a byte array?  Seems like all instances are the same

Reply:
The byte array for the key is found here: https://github.com/vincentkam/libmongocrypt/blob/csharp/bindings/cs/MongoDB.Crypt/LocalKmsCredentials.cs#L44
The class itself currently does nothing, but in a different branch, all implementations of `IKmsKeyId` contain `AlternateKeyNames` an sequence of BsonDocuments converted to byte arrays

Reply:
I still don't understand what this class is for.

Reply:
Each `Id` class is paired with a `Credentials` class.  Each `Id` class has a `SetCredentials` method that is responsible for calling `mongocrypt_ctx_setopt_masterkey_[local/aws]`. Each `Credentials` class has a `SetCredentials` method responsible for calling mongocrypt_setopt_kms_provider_[aws/local]`. Keeping the classes separate makes each class responsible for only calling one method (ignoring alt key name support).